Sources/EpoxyBars/BottomBarInstaller/BottomBarContainer.swift

```diff
@@ -66,6 +66,9 @@ public final class BottomBarContainer: BarStackView, InternalBarContainer {
     }
   }
 
+  /// The `BottomBarInstaller` that manages this `BottomBarContainer`
+  public internal(set) weak var barInstaller: BottomBarInstaller?
```

Comment:
Can we add a parallel reference for the `TopBarInstaller` while we're here? Just for symmetry between the two

Reply:
sure

Reply:
Updated 👍🏻